security/pfSense-pkg-snort/files/usr/local/www/snort/snort_rulesets.php

```diff
@@ -44,29 +39,31 @@
         exit;
 }
 
-if (isset($id) && $a_nat[$id]) {
-	$pconfig['enable'] = $a_nat[$id]['enable'];
-	$pconfig['interface'] = $a_nat[$id]['interface'];
-	$pconfig['rulesets'] = $a_nat[$id]['rulesets'];
-	if (empty($a_nat[$id]['autoflowbitrules']))
+$a_nat = config_get_path("installedpackages/snortglobal/rule/{$id}", []);;
+
+if (isset($id) && $a_nat) {
+	$pconfig['enable'] = $a_nat['enable'];
+	$pconfig['interface'] = $a_nat['interface'];
+	$pconfig['rulesets'] = $a_nat['rulesets'];
+	if (empty($a_nat['autoflowbitrules']))
 		$pconfig['autoflowbitrules'] = 'on';
 	else
-		$pconfig['autoflowbitrules'] = $a_nat[$id]['autoflowbitrules'] == 'on' ? 'on' : 'off';;
-	$pconfig['ips_policy_enable'] = $a_nat[$id]['ips_policy_enable'] == 'on' ? 'on' : 'off';;
-	$pconfig['ips_policy'] = $a_nat[$id]['ips_policy'];
-	$pconfig['ips_policy_mode'] = $a_nat[$id]['ips_policy_mode'];
+		$pconfig['autoflowbitrules'] = $a_nat['autoflowbitrules'] == 'on' ? 'on' : 'off';;
+	$pconfig['ips_policy_enable'] = $a_nat['ips_policy_enable'] == 'on' ? 'on' : 'off';;
```

Comment:
More extra semicolons.

Reply:
Can you say "copy and paste" an extra semicolon <grin>. Removed the extra one.